### ISSUE TITLE
[Multisite] Error to send new post since WP 7.9 / 7.9.1 / 8.0

### ISSUE THREAD

**Issue by LiloBzH** [wordpress-mobile/WordPress-Android#6484]
##### Tested on [device], Android [version]
Android 7 and 6
Wordpress Android 7.9, 7.9.1 , 8.0 => NOK
Wordpress 7.8 ==>> OK !

Error when post new article

My conf : Multisite + htaccess auth

![screenshot_20170806-214440](https://user-images.githubusercontent.com/4649466/29006911-5cc74494-7af9-11e7-9018-f32fb4114bed.png)

**Comment by LiloBzH** [comment]
Same thing with 7.9.1 version ...

**Comment by LiloBzH** [comment]
Error is here : 

wp-includes/class-wp-xmlrpc-server.php:1455

if ( ! $term )
							return new IXR_Error( 403, __( 'Invalid term ID.' ) );

**Comment by LiloBzH** [comment]
So, wordpress dev are really bad, meanwhile, here is a solution

For moment , to solve the problem 

Add in file wp-includes/class-wp-xmlrpc-server.php 

Line 1453 
if ( ! $term ) { $term = 1; }

I dont know why but sometimes, post has not term so Error !
So, if there is no term, we force the term 1 (unclassed term)

**Comment by daniloercoli** [comment]
Hey @LiloBzH - This issue should have been fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/6520

Would you mind to try `release/8.0`?

**Comment by LiloBzH** [comment]
it's not fix in the 8.0 version

Same error

**Comment by daniloercoli** [comment]
Are you trying to upload a post that has some categories attached to it? 
```
if ( ! $term )
	return new IXR_Error( 403, __( 'Invalid term ID.' ) );
```
Could happen when the app sends a list of categories attached to the post, but one of those are not available on the server.

**Comment by LiloBzH** [comment]
I tested with 2 categories checked => same error.

$term seems to be empty for some unknown reason

**Comment by oguzkocer** [comment]
@LiloBzH Could you please update the app to 8.1 and see if the issue is resolved for you? Thanks.

**Comment by LiloBzH** [comment]
Version 8.1 is not available at this time on google store. I'm waiting the update

**Comment by oguzkocer** [comment]
It's rolling out and should be available throughout the day.

**Comment by LiloBzH** [comment]
Hi !

it's ok with wordpress 8.1. Thank's for the update

**Comment by oguzkocer** [comment]
Thanks for following up on it @LiloBzH!

Fixed via #6520.
